src/components/NotFoundPage/NotFoundPage.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+import { Button } from 'reactstrap';
+
+import styles from './NotFoundPage.module.css';
+
+const NotFoundPage = ({currentPosition}) => {
+        return (
+            <div className={styles.container}>
+                <div className="error-title">
+                    <h2>Error 404</h2>
+                    <h4>Page Not Found</h4>
+                    <p>
+                        The resource you are trying to access could not be found. 
```

Comment:
minor thing, but if you get the change can you change this from "resource" to page? or something. the word "resource" has a specific meaning in this app